src/main/resources/application.properties

```diff
@@ -18,14 +18,11 @@ spring.datasource.initialization-mode=always
 meetup.api.url=https://api.meetup.com/rheinjug
 
 # --- MINIO ---
-# Minio Host
-#spring.minio.url=http://localhost:9001
-# Minio Bucket name for your application
-#spring.minio.bucket=rheinjug
-# Minio access key (login)
-#spring.minio.access-key=minio
-# Minio secret key (password)
-#spring.minio.secret-key=minio123
+#minio.host=${MINIO_HOST:http://localhost}
```

Comment:
Diese Application Properties sind tatsächlich sinnvoll, da könnt ihr in den Services die Properties mit @Value rausziehen